src/builtin/tables.scrbl

```diff
@@ -29,12 +29,32 @@
 
 @docmodule["tables" #:noimport #t #:friendly-title "Tables"]{
 
-  Tables consist of a sequence of zero or more @italic{rows}, which each
-  contain an equal amount of entries in named @italic{columns}.
-
+There are many examples of tables in computing, with
+spreadsheets being the most obvious.
+                                                             
+A @pyret{Table} is made up of @bold{rows} and @bold{columns}. Each
+column has a name.  Each row has an equal number of columns,
+in the same order.  Each column may be assigned a type via an
+annotation, and each entry in that column will be checked against
+the annotation.
+
+@margin-note{Note that as of June 2017, the @pyret{tables} implementation in
+is rather minimal.  It meets the basic requirements of the curricula that
+use it, but lacks many features that one might reasonably expect from a
+complete table library.
+
+Additionally, after the first year of using @pyret{tables}, the Pyret team
+is considering a new table design and interface that will make it easier
+to add features in the future, but in the meantime puts small changes
+to the current design on hold.
+
+Tabular data is here to stay in Pyret and Pyret-related curricula, so
+count on further development of @pyret{tables}.}
+  
   @section[#:tag "s:tables"]{Creating Tables}
 
-Tables are created with a @pyret{table:} expression, which lists any number of
+A simple @pyret{Table} can be "manually" created with a @pyret{table:}
```

Comment:
I'd say `can be directly created` instead of `can be "manually" created` -- there's no need for scare-quotes; table expressions are perfectly fine, if tedious, ways to build table values.